incubator/hnc/Makefile

```diff
@@ -30,17 +39,38 @@ test: build
 # Builds all binaries (manager and kubectl) and manifests
 build: generate fmt vet manifests
 	go build -o bin/manager ./cmd/manager/main.go
-	go build -o bin/kubectl-hnc ./cmd/kubectl/main.go
+	go build -o bin/kubectl/kubectl-hnc ./cmd/kubectl/main.go
 
+# Clean all binaries (manager and kubectl)
+clean:
+	rm -r bin/kubectl/kubectl-hnc
+	rm -f bin/manager
+	
 # Install kubectl plugin
 kubectl: build
-	go build -o ${GOPATH}/bin/kubectl-hnc ./cmd/kubectl/main.go
+	cp bin/kubectl/kubectl-hnc ${GOPATH}/bin/kubectl-hnc
 	@echo "Installed kubectl-hnc to GOPATH/bin"
 
 # Run against the configured Kubernetes cluster in ~/.kube/config
 run: build
 	go run ./cmd/manager/main.go --novalidation
 
+# Install kubectl plugin locally using krew.
+krew-build: build tar
+	sed -i 's/^\(\s*sha256\s*:\s*\).*/\1"$(KREW_CKSM)"/' \
+		hack/hnc.yaml
+	sed -i 's/^\(\s*version\s*:\s*\).*/\1"$(VERSION)"/' \
+		hack/hnc.yaml
+	kubectl krew install --manifest=hack/hnc.yaml --archive=hack/hnc.tar.gz
+
+# Make krew archive and put into /hack
+tar:
+	tar -zcvf hack/hnc.tar.gz bin/kubectl
```

Comment:
`hack` is a Git folder, whereas `bin` is not. Can we create the tar file in `bin` instead of `hack`?

Also, `hnc.tar.gz` isn't very descriptive because _everything_ in this (part of the) repo is HNC. So how about `kubectl-hnc.tar.gz` instead?

Reply:
done